lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -119,3 +116,29 @@ void migrateClassToMixin(ClassDeclaration node, YieldPatch yieldPatch,
   propsAndStateClassNamesConvertedToNewBoilerplate[originalPublicClassName] =
       newMixinName;
 }
+
+class SemverHelper {
+  Map _exportList;
+
+  SemverHelper(Map jsonReport) {
+    _exportList = jsonReport['exports'];
+  }
+
+  /// Returns semver report information for [node] if it is publicly exported.
+  ///
+  /// If [node] is not publicly exported, returns `null`.
+  Map<String, dynamic> getPublicExportLocations(ClassDeclaration node) {
+    final className = node?.name?.name;
+    String classKey;
+
+    if (className == null || _exportList == null) return null;
```

Comment:
It seems like we may want this to be a louder failure. Have we seen the class name be `null` before? I'd be curious to see what that code looks like haha

If these can be `null` though, I'd expect it to throw an `Exception` since we need the list and class name to really verify if the class is exported, and returning `null` could be a false negative